smart-contracts/cookbook/smart-contracts/deploy-nft/nft-remix.md

```diff
@@ -24,62 +24,46 @@ This guide demonstrates how to deploy an [ERC-721](https://eips.ethereum.org/EIP
 
 Navigate to [Remix](https://remix.ethereum.org/){target=\_blank} in your web browser.
 
-The interface will load with a default workspace containing sample contracts. In this interface, you can access a file explorer, edit your code, interact with various plugins for development, and use a terminal.
+The interface will load with a default workspace containing sample contracts. In this interface, you can access a file explorer, edit your code, interact with various development plugins, and use a terminal.
```

Comment:
Closest I could get to it, but can you please compare the prereqs section with the ERC-20 Remix guide and make sure they look the same?

Reply:
Also, there are some inconsistencies with this Access Remix section. We have it for the Deploy a Basic Contract Remix guide (but it has an image there and not here), and we don't have it for the Deploy an ERC-20 Remix guide